index.html

```diff
@@ -15,12 +15,45 @@
     <link rel="stylesheet" href="css/layout.css" />
     <link rel="stylesheet" href="css/styles.css" />
   </head>
+
   <body>
     <header>
-      <div class="container"></div>
+      <div class="container">
+        <div class="row">
+          <h1>Code Journal</h1>
+        </div>
+      </div>
     </header>
     <main class="container">
-      <div data-view="entry-form"></div>
+      <form action="submit" id="entry-form" method="post">
+        <div data-view="entry-form">
```

Comment:
These should be swapped. The `form` should be inside the `div` with the `data-view="entry-form"`